docs/tutorial/image_det.md

```diff
@@ -1,3 +1,128 @@
 # Training for Object Detection
 
-Object detection is in preparation.
+This guide trains a neural network model to object detection of Human Face and Human Hand in the [Open Images Dataset V4](https://storage.googleapis.com/openimages/web/index.html) on GPU server.
+
+<img src="../_static/openimages_v4.png" width="600">
+
+## Preparation
+
+The Open Images V4 dataset is available from official website, but original dataset is too large for the tutorial. We provide reduced dataset.
+
+Blueoil supports 2 formats for object detection.
+
+- OpenImagev4 format
+- DeLTA-Mark format
+
+Note: *Please see the detail in <a href="../usage/dataset.html">Prepare training dataset</a>*
+
+You can download subset of Open Images V4 from
+[our server](https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages.tgz).
+
+
+	$ wget https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages.tgz
+	$ tar xzf openimages.tgz
+
+
+This dataset consists of 5,000 color images in 2 classes, with 2,500 images per class.
+
+## Generate a configuration file
+
+Generate your model configuration file interactively by running `blueoil init` command.
+
+    $ ./blueoil.sh init
+
+This is an example of configuration.
```

Comment:
Command is "./blueoil.sh init". But there are no words like "initialize". 
In the first place, "init" may not be appropriate.

Reply:
For example, we write like below.
This is an example of initialization (or add some modifier).